src/cmd_start.rs

```diff
@@ -289,7 +292,16 @@ pub fn command_start< 'a >( matches: &clap::ArgMatches< 'a > ) -> Result< (), Er
         }
 
         debug!( "Received a request for {:?}", path );
-        if let Some( mut artifact ) = last_build.deployment.get_by_url(&path) {
+
+        let mut artifact = last_build.deployment.get_by_url( &path );
+        if artifact.is_none() {
+            if let Some( ref not_found_path ) = path_404 {
+                println!("Artifact could not be found, but should try to get user specified path instead");
```

Comment:
Having this print here is probably unnecessary (it'll just spam the console); you can put a `debug!` log instead, e.g.:

```
debug!( "{} not found; serving {} instead", path, path_404 );
```

Reply:
Yeah, that was there as an accident (switched back from Clion to VS Code, and forgot that autosave was disabled when I made the commit).

Reply:
Still a good idea to add the debug though.